mythril/support/loader.py

```diff
@@ -36,6 +43,9 @@ def read_storage(self, contract_address, index):
 
     def dynld(self, contract_address, dependency_address):
 
+        if not self.contract_loading:
+            raise ValueError("Cannot load contract when contract_loading flag is false")
```

Comment:
Why a value error here, and an Exception a while back?
In this case a specific exception might make sense, wdyt?

Reply:
This exception is being caught in lot of other areas in laser, so we might need to import this exception from mythril to laser, should we do that?